lm_eval/tasks/__init__.py

```diff
@@ -26,6 +26,9 @@
     QMSum,
 )
 
+from lm_eval.caching.cache import save_to_cache, load_from_cache
```

Comment:
This file is getting a pretty heavy rewrite in #1321 -- would you be willing to check out shifting these changes to that once that settles?

#1321 should speed up task loading / registration but caching them would still be great.

Reply:
Yeah for sure! np np. 

Only pushback would be that users who could benefit from the caching would be missing out on it in the interim, so if it's going to be a while, I'd lean towards us integrating the current changes, but if not, then strong agree 👍👍, def seems like it'll be a merge pain

Reply:
Let me get back to you on the timeline for that PR—think we were originally hoping to merge it this week but will check on that.

Reply:
We've merged that PR, as it should speed up task registry at initialization quite a bit in the meantime!

Reply:
Sick! Yeah, I'll slice off a chunk of time this weekend and sharpen this all up